### DISCUSSION TITLE
Inline tags separator recognition

### DISCUSSION THREAD

**Discussion by acdesigns** [RafaelGB/obsidian-db-folder#804] [Q&A]
### Contact Details

ac.designs@gmail.com

### What happened?

The plugin only recognises tags if they are in the YAML or inline with tags::, this is fine, I can work with that.

HOWEVER, with inline tags:: All tags are seen as one tag within DB plugin. Each tag is preceded with "#"as well as seperated with a ","

Is there a way to have DB plugin see each as a separate tag?
<img width="521" alt="Screenshot 2023-01-30 at 2 10 46 PM" src="https://user-images.githubusercontent.com/59508042/215572452-7f4f9185-742d-45a3-8f85-2fcf22122ea5.png">

### What platform were you using?

Desktop

### Version of the plugin

3.3.0

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
Hello @acdesigns ,
It should be recognised as a separate tag once you include a comma. I will test your case

**Comment by acdesigns** [comment]
> Hello @acdesigns , It should be recognised as a separate tag once you include a comma. I will test your case

I thought it would too. You can see in the screenshots it doesn't. Thank you for adding it to what I'm sure is a long list of things you want to do eventually. Thank you for your hard work on this, it's an amazing plugin already :)

**Comment by RafaelGB** [comment]
I just try to test this issue and I can not reproduce the problem.

I just can force it with quotes:
- "#tag1, #tag2", #tag3. => generates 2 pills instead of three for example

**Reply by acdesigns** [reply]
Alright. I've decided to go another route, using a dataview table to list the information I want, and the syntax doesn't work with your plugin anyway. It would have been nice for getting a bird's eye view of the data, but I'll figure out something else :)

Thank you for trying!
